.travis.yml

```diff
@@ -19,6 +19,11 @@ cache:
     - $HOME/.npm
 
 before_install:
+  - |
+    git diff --name-only $TRAVIS_COMMIT_RANGE | grep -qvE '(\.md$)' || {
+      echo "Only docs were updated, stopping build process."
```

Comment:
Thanks for adding the output! That'll make it easy to diagnose why, in some wild future where we want to lint Markdown or something in a test, why they aren't running 😄

Reply:
NIce, and on that subject, I do want to lint Markdown in the future, and the above can be leveraged so that again if only *.md files were changed then only lint the Markdown files 👌